content/influxdb/cloud/reference/internals/durability.md

```diff
@@ -0,0 +1,79 @@
+---
+title: InfluxDB Cloud data durability
+description: >
+  InfluxDB Cloud ensures the durability of all stored data by replicating data across
+  geographic locations, automatically creating backups, and verifying that replicated
+  data is consistent and backups are readable.
+weight: 101
+menu:
+  influxdb_cloud_ref:
+    name: Data durability
+    parent: InfluxDB Cloud internals
+influxdb/cloud/tags: [backups, internals]
+---
+
+InfluxDB Cloud ensures the durability of all stored data by replicating data across
+geographic locations, automatically creating backups, and verifying that replicated
```

Comment:
Do we replicate across geographic locations? I think we do replicate data over availability zones within a specific region - and they are described by AWS as "Availability Zones are multiple, isolated locations within each Region." so perhaps that counts, but I wanted to flag it.

Reply:
InfluxDB replicates data in the storage tier across...

Reply:
because the LP data is stored in an object store location, it is available for recovery even if the entire region were to disappear.